libinstall/CentWeb.sh

```diff
@@ -226,16 +251,20 @@ ${CAT} "$file_sql_temp" | while read file ; do
 done
 check_result $flg_error "$(gettext "Change macros for sql update files")"
 
-### Step 2.0: Change right on Centreon WebFront
+### Step 2.0: Modify rights on Centreon WebFront and replace macros
+
+## create a random APP_SECRET key
+HEX_KEY=($(dd if=/dev/urandom bs=32 count=1 status=none | $PHP_BIN -r "echo bin2hex(fread(STDIN, 32));"))
+log "INFO" "$(gettext "Generated a random key") : $HEX_KEY"
 
 ## use this step to change macros on php file...
-macros="@CENTREON_ETC@,@CENTREON_CACHEDIR@,@CENTPLUGINSTRAPS_BINDIR@,@CENTREON_LOG@,@CENTREON_VARLIB@,@CENTREONTRAPD_BINDIR@"
+macros="@CENTREON_ETC@,@CENTREON_CACHEDIR@,@CENTPLUGINSTRAPS_BINDIR@,@CENTREON_LOG@,@CENTREON_VARLIB@,@CENTREONTRAPD_BINDIR@,@PHP_BIN@"
 find_macros_in_dir "$macros" "$TMP_DIR/src/" "www" "*.php" "file_php_temp"
-
-log "INFO" "$(gettext "Apply macros")"
+find_macros_in_dir "$macros" "$TMP_DIR/src/" "bin" "*" "file_bin_temp"
+log "INFO" "$(gettext "Apply macros on php files")"
 
 flg_error=0
-${CAT} "$file_php_temp" | while read file ; do
+${CAT} "$file_php_temp" "$file_bin_temp" | while read file ; do
     log "MACRO" "$(gettext "Change macro for") : $file"
```

Comment:
log "MACRO" seems odd.

Reply:
it's only a param to specify in the logs what it is about.
The installation log can be found in the centreon's source folder,
eg : centreon-web-19.10.6/log/install_centreon.log
Here how it's processed : https://github.com/centreon/centreon/blob/master/libinstall/functions#L121
Do you want this param value to be changed to something more consistent ?
![image](https://user-images.githubusercontent.com/34628915/74729848-68a3ae80-5245-11ea-85fe-52efa6e6ee2f.png)

Reply:
I think it's meant to be INFO.